### DISCUSSION TITLE
Width of the images

### DISCUSSION THREAD

**Discussion by diamuxin** [retypeapp/retype#255] [General]
Hello, is it possible to modify the width-height of the images in .md documents? 
Thanks & BR.

**Comment by AndreyChechel** [comment]
Hi @diamuxin 

Thanks for interest in Retype.

Unfortunately there is no way to configure image size using standard Markdown inline syntax: `![image caption](/path/to/image)`. However, you can achieve similar result using the [embed](https://retype.com/components/embed/) component. Please make sure you set [`el`](https://retype.com/components/embed/#el) attribute to either `embed` or `object` type.

```md
[!embed el="embed" width="100"](/static/instagram.png)
``` 

Also, as a fallback option you can always use raw HTML. The only drawback is that Retype will not resolve the `src` file location, so the `src` value needs to be exactly the same as it's expected to be in the final page markup.

```html
<img src="/static/instagram.png" width="100" />
```

**Reply by geoffreymcgill** [reply]
Setting a custom width on a markdown image element is now supported using the generic attribute syntax. For example, the following sample demonstrates setting a custom `250px` width on an image:

```md
![](example.jpg){width="250px"}
```

The functionality is available now in Retype [`v2.0.0-preview`](https://github.com/retypeapp/retype/discussions/287#discussioncomment-2321442) and should be generally available next week with the final `v2.0.0` release.

Hope this helps.

**Reply by nembal** [reply]
I just tested this, and the `{width="250px"}` and it does not work
